docs/UserGuide.md

```diff
@@ -188,9 +188,10 @@ Format: `find n/NAME p/PHONE r/ROOM_NUMBER t/TAG`
   e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
 * The order of the `TAGS` does not matter e.g. `t/friends t/colleagues` works the same
   as `t/colleagues t/friends`
-* `TAG` search is case-sensitive e.g. `Friends` will not match `friends`
-* When searching by `TAG`, the contact must match every `TAG` in the input fields to appear in the listed results.
-* If the user inputs multiple fields for the `find`, then only contacts that match all the input fields will appear in the listed results.
+* Tag search is case-insensitive, `Friends` will match `friends`
+* In tag search, the contacts have to contain all the specified TAGs in order to match
```

Comment:
remember to add full stops for lines 191 and 192

Reply:
Perhaps we can rephrase line 192 to something more intuitively understood? 

"The results of tag search will only contain the contacts that match all of the specified tags.